web/client/reducers/queryform.js

```diff
@@ -479,9 +480,12 @@ function queryform(state = initialState, action) {
             return {...state, simpleFilterFields: []};
         }
         case LOAD_FILTER:
-            const {spatialField, filterFields, groupFields, crossLayerFilter} = (action.filter || initialState);
+            const {spatialField, filterFields, groupFields, crossLayerFilter, attributePanelExpanded, spatialPanelExpanded, crossLayerExpanded} = (action.filter || initialState);
             return {...state,
                     ...{
+                    attributePanelExpanded,
+                    spatialPanelExpanded,
+                    crossLayerExpanded,
                     spatialField: {
                         ...spatialField,
```

Comment:
Thesting, seems that "attribute" is not restored correctly.
In fact the filter is created using "the_geom". Before this pull request attribute was filled by some triggered describeFeature (both for featuregrid or widgets) to set the correct spatial attribute. 

Make you sure that: 
 - spatialField geometry attribute name is correctly set for featuregrid, new widget and saved widget (during a second editing)
 - Also the crossLayerFilter has attribute value that is syncronized with the same logic. Make you sure that also this attribute is correctly set in any case of the ones listed above.  


**Add some unit tests to prove that all these things work**.